src/storage/hummock_sdk/src/compaction_group/hummock_version_ext.rs

```diff
@@ -695,9 +701,29 @@ impl HummockLevelsExt for Levels {
                     "should find the level to insert into when applying compaction generated delta. sub level idx: {},  removed sst ids: {:?}, sub levels: {:?},",
                     insert_sub_level_id, delete_sst_ids_set, l0.sub_levels.iter().map(|level| level.sub_level_id).collect_vec()
                 );
+                if l0.sub_levels[index].table_infos.is_empty()
+                    && self.member_table_ids.len() == 1
+                    && insert_table_infos.iter().all(|sst| {
+                        sst.table_ids.len() == 1 && sst.table_ids[0] == self.member_table_ids[0]
```

Comment:
add comments

Reply:
sub_levels should not be empty in this branch？That means `l0.sub_levels[index].table_infos.is_empty()` is always be false

Reply:
done